extensions-contrib/prometheus-emitter/src/test/java/org/apache/druid/emitter/prometheus/PrometheusEmitterConfigTest.java

```diff
@@ -48,4 +48,82 @@ public void testEmitterConfigWithBadExtraLabels()
     Assert.assertTrue(actualMessage.contains(expectedMessage));
   }
 
+  @Test
+  public void testDefaultConstructor()
+  {
+    PrometheusEmitterConfig config = new PrometheusEmitterConfig(null, null, null, null, null, false, false, null, null, null, null);
+    Assert.assertEquals(PrometheusEmitterConfig.Strategy.exporter, config.getStrategy());
+    Assert.assertEquals("druid", config.getNamespace());
+    Assert.assertNull(config.getDimensionMapPath());
+  }
+
+  @Test
+  public void testExporterStrategy()
+  {
+    PrometheusEmitterConfig config = new PrometheusEmitterConfig(PrometheusEmitterConfig.Strategy.exporter, "druid", null, 8080, null, true, true, null, null, null, null);
+    Assert.assertEquals(PrometheusEmitterConfig.Strategy.exporter, config.getStrategy());
+    Assert.assertEquals("druid", config.getNamespace());
+    Assert.assertEquals(8080, config.getPort());
+    Assert.assertTrue(config.isAddHostAsLabel());
+    Assert.assertTrue(config.isAddHostAsLabel());
+  }
+
+  @Test
+  public void testExporterStrategyWithoutPort()
+  {
+    Assert.assertThrows(IllegalArgumentException.class, () -> {
+      new PrometheusEmitterConfig(PrometheusEmitterConfig.Strategy.exporter, null, null, null, null, false, false, null, null, null, null);
+    });
+  }
```

Comment:
Also combined these two tests into one, so that both ExporterStrategy and PushGatewayStrategy will have one test each.